packages/blocks/src/attachment-block/utils.ts

```diff
@@ -212,6 +214,8 @@ export function isAttachmentLoading(modelId: string) {
 
 /**
  * Use it with caution! This function may take a long time!
+ *
+ * @deprecated
```

Comment:
Since the `hasBlob` is similar to `getAttachmentBlob`, I have migrated to `getAttachmentBlob`.

But neither of them is a best practice. the `BlobManager` needs to be refactored to meet the following requirements:

- A new `head` API, also known as `has`, needs to be added to check a blob.
- The `get` and `set` methods should return an Object that contains a stream to make it easier to check the progress of the upload or download.
- The `list` API should have a default maximum limit and the ability to be adjusted by its options.

I think the [cloudflare R2 workers API](https://developers.cloudflare.com/r2/api/workers/workers-api-reference/#bucket-method-definitions) is a good reference.